src/ripple/shamap/SHAMap.h

```diff
@@ -213,6 +213,10 @@ class SHAMap
     const_iterator
     upper_bound(uint256 const& id) const;
 
+    // traverse functions
+    const_iterator
+    lower_bound(uint256 const& id) const;
```

Comment:
We should document what these functions do. I know there's an existing almost identical function here that isn't documented. Still, I think new functions should include a a brief description of what it does.

Ditto for the new `lastBelow` function.